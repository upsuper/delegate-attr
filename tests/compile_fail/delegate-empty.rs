use delegate_attr::delegate;

struct Foo(Vec<u8>);

#[delegate]
impl Foo {
    fn len(&self) -> usize {}
}

fn main() {}
