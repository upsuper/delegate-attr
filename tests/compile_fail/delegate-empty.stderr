error: unexpected end of input, expected expression
 --> $DIR/delegate-empty.rs:5:1
  |
5 | #[delegate]
  | ^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
